src/plugins/express.js

```diff
@@ -9,6 +9,8 @@ const pathToRegExp = require('path-to-regexp')
 const OPERATION_NAME = 'express.request'
 
 function createWrapMethod (tracer, config) {
+  const recordHeaders = config.recordHeaders ? config.recordHeaders.map(key => key.toLowerCase()) : []
```

Comment:
Make sure it's an array first to be safe.

Reply:
I figured that an error here, early in the process, could help setup of the plugin. What would you want to happen if this is not an array?

Reply:
The idea is to try and never crash, even if any user input is wrong. In any case where it would be interesting to know why something is not working, a log entry of the appropriate level should be created instead, which the user will be able to view in debug mode.